### ISSUE TITLE
[bug]: Web version keeps saying "Disconnected"

### ISSUE THREAD

**Issue by enzyme69** [invoke-ai/InvokeAI#1315]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

macOS

### GPU

mps

### VRAM

8GB

### What happened?

When generating, and take a while like 15 minutes, the WEB is saying "Disconnected"/

**Comment by psychedelicious** [comment]
Hi @enzyme69 , Please give us some more details:
- Which branch are you on?
- Are you running txt2img, img2img, inpainting, or postprocessing?
- Are there any errors in the terminal window?
- What size of image are you attempting to work on?

**Comment by ctalibard** [comment]
@psychedelicious I see disconnections when running inpainting on images over a certain size. For example if I 

1. Txt2img "still life of apples on a table" at 768 x 512. 
2. Upscale that by 4x 
3. Send to inpainting
4. set a 512 x 512 bounding box around one of the apples along with a mask
5. inpaint "a worm coming out of an apple"

... then it disconnects and reconnects instantly but nothing happens.

My suspicion is that the Base64 version of the image gets large enough push the message pass the maximum size allowed by socket.io so the server terminates the connection. I tried increasing the limit in `backend/invoke_ai_web_server.py` but it didn't seem to help. At this point I would add my expertise lies more in React than Python so take that with a pinch of salt. 

I suspect this is a different issue to the once mentioned above because the server is clearly not even starting to process the request. Happy to raise a separate issue for this if you want.

**Comment by enzyme69** [comment]
@psychedelicious 

- main branch I think
- txt2img
- I did not see any error in terminal. 
- 512 x 768

This disconnection happens in M1 8GB and old MacBookPro Intel. Maybe because processing was too slow, it disconnects itself and sometimes it's possible to get connection back.

When disconnected, I cannot stop the Invoke, however the Terminal processing is still happening.

In some cases my Safari just crashes itself (over night). The processing in command line is still running however.

**Comment by psychedelicious** [comment]
I think there is nothing wrong, just how the current server works. The server is not multithreaded so if generation takes a long time the websocket connection gets timed out. 

As you saw, the actual processing continues. Once processing is finished you should be able to refresh the web ui and continue where you left off.

We will be migrating to a more capable server relatively soon which will fix this kind of issue.
